x-pack/plugins/security_solution/common/detection_engine/schemas/request/create_rules_type_dependents.ts

```diff
@@ -107,6 +107,24 @@ export const validateThreshold = (rule: CreateRulesSchema): string[] => {
   return [];
 };
 
+export const validateThreatMapping = (rule: CreateRulesSchema): string[] => {
+  let errors: string[] = [];
+  if (rule.type === 'threat_match') {
```

Comment:
Nit: saw that I think one of the more recent PRs added the pattern of using a helper function to check rule type (`x-pack/plugins/security_solution/common/detection_engine/utils.ts`) like on line 98 above - maybe a threat match one can be added.

Reply:
Actually yeah I had one in the backend on the server side but it looks like these have moved to the common section so totally agree. I am moving the server side one to the common section and then collapsing code and cleaning this one up. Thanks for the 🦅   👀 on these.